public/index.html

```diff
@@ -295,22 +294,11 @@
     <!-- FOOTER: DESCRIPTION AND STATUS -->
     <footer class="navbar-fixed-bottom">
```

Comment:
This footer is also not as it should be.

     <footer class="navbar-fixed-bottom infoArea row margin-0">
        <div class="col-xs-12 col-sm-8 col-md-8 col-lg-8 gameLogWindow">
            <ul class="gameLog"></ul>
        </div>
        <div class="col-xs-12 col-sm-4 col-md-4 col-lg-4 characterStatusWindow">
          <p>Status Window</p>
        </div>
      </div>
    </footer>